docs/can-canjs/canjs.md

```diff
@@ -32,7 +32,7 @@ h3 {
   color: #282C33;
   margin-bottom: 10px;
 }
-code:before {
+code[class*=language-]:before {
```

Comment:
This fixes the issue, but I think the bigger issue is that all these styles apply to the rest of the page.

I think we should scope all these styles to `.body` so they only apply to the homepage.